src/lib/components/form/BuildForm.svelte

```diff
@@ -183,6 +187,12 @@
       saving = false;
     }
   }
+
+  function filterTalents(talent: Item[] | Power[]): boolean {
+    // Filter on the full object by stringifying it. Bit ugly, but that makes it easy to filter by name,
+    // description, and stat names all at once.
+    return JSON.stringify(talent).toLowerCase().includes(query.toLowerCase());
```

Comment:
Technically I think this means searching any field name would highlight everything? Ah well, to be fixed later I suppose.

Reply:
It does, I figured it doesn't matter for now